src/modelbench/modelgauge_runner.py

```diff
@@ -53,12 +64,11 @@ class ModelGaugeSut(SutDescription, Enum):
     )
     VICUNA_13B = "vicuna-13b", "LM Sys Vicuna v1.5 (13B)", TogetherChatSUT, "lmsys/vicuna-13b-v1.5"
     WIZARDLM_13B = "wizardlm-13b", "WizardLM v1.2 (13B)", TogetherChatSUT, "WizardLM/WizardLM-13B-V1.2"
-    # YI_34B_CHAT = "yi-34b", "01-ai Yi Chat (34B)", TogetherChatSUT, "zero-one-ai/Yi-34B-Chat"
+    # YI_34B_CHAT = "yi-34b", "01-ai Yi Chat (34B)", IntegratedTogetherChatSUT, "zero-one-ai/Yi-34B-Chat"
```

Comment:
What/where is `IntegratedTogetherChatSUT`?

Reply:
Ah, sorry. That is now TogetherChatSUT again. I first tried this under a separate name, as I hadn't yet kicked out the existing plugins from pyproject.toml. I'll fix that.